src/components/Breadcrumb/Breadcrumb.utils.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ReactElement } from 'react'
+
+import { BREADCRUMB_COLLAPSED_WIDTH, BreadcrumbCollapsed } from './Breadcrumb.Collapsed'
+import { BREADCRUMB_SEPARATOR_PADDING, BREADCRUMB_SEPARATOR_SIZE } from './Breadcrumb.Separator'
+import { BreadcrumbButton, BreadcrumbItemMenuItem, BreadcrumbItemType } from './Breadcrumb.types'
+import { BreadcrumbItem } from './Breadcrumb.Item'
+
+export const renderItem = (
```

Comment:
No real reasons, I simply prefer it like that. It should not have an impact either way

Reply:
I understand, making it a react component would allow us to use it with jsx syntax and to use hooks (e.g. for memoization) in case of need 🤔  

btw either way is fine, you can resolve this discussion if don't find it useful :)